influxdb/2.0/alpine/entrypoint.sh

```diff
@@ -29,17 +30,18 @@ function log () {
         shift 2
     done
 
-    local logtime=$(date --utc +'%FT%T.%NZ')
+    local logtime="$(date --utc +'%FT%T.%NZ')"
     1>&2 echo -e "${logtime}\t${level}\t${msg}\t{${attrs}}"
 }
 
 # Set the global log-level for the entry-point to match the config passed to influxd.
 function set_global_log_level () {
-    local level="$(influxd print-config --key-name log-level ${@})"
+    local level="$(influxd print-config --key-name log-level "${@}")"
     if [ -z "$(log_level ${level})" ]; then
```

Comment:
This will never be false, right? `log_level` always returns a string.

Reply:
Good catch (mirrored the non-alpine version, but didn't clean up)